package.json

```diff
@@ -59,6 +59,7 @@
     "jest": "28.0.3",
     "lint-staged": "^13.0.1",
     "prettier": "^2.3.2",
+    "prisma": "^3.15.1",
```

Comment:
Do we still need prisma?

Reply:
Should we remove it from this card, or open another card to do it?

Reply:
remove it in this card

Reply:
I have already removed it.